Cargo.toml

```diff
@@ -43,9 +43,10 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 provwasm-std = { version = "1.0.0-beta" }
 cosmwasm-std = { version = "1.0.0-beta" }
 cosmwasm-storage = { version = "1.0.0-beta" }
-cw-storage-plus = "0.8.0"
-cw2 = "0.8.1"
+cw-storage-plus = "0.12.1"
+cw2 = "0.12.1"
 schemars = "0.8.3"
+semver = "1"
```

Comment:
Cosmwasm libs recommended converting contract versions to SemVer for comparisons.  Given that it grants us immediate access to ordinal values, I think it's pretty slick